{"name": "test_vwo_login_positive", "status": "broken", "statusDetails": {"message": "AttributeError: 'WebDriver' object has no attribute 'find_element_by_id'", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"c6a5b4f7960898b34ed822284d7108c3\")>\n\n    @allure.epic(\"VWO Login Test\")\n    @allure.feature(\"TC#1 - VWO App Positive Test\")\n    @pytest.mark.positive\n    def test_vwo_login_positive(setup):\n        driver = setup\n        loginPage = LoginPage(driver)\n>       loginPage.login_to_vwo(username=\"contact+atb5x@thetestingacademy.com\", password=\"Wingify@SDET\")\n\ntest_vwo_login.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <tests.pageObjects.loginPage.LoginPage object at 0x0000018EADB8AD90>\nusername = 'contact+atb5x@thetestingacademy.com', password = 'Wingify@SDET'\n\n    def login_to_vwo(self, username, password):\n>       self.driver.find_element_by_id(self.get_email_address()).send_keys(username)\nE       AttributeError: 'WebDriver' object has no attribute 'find_element_by_id'\n\n..\\pageObjects\\loginPage.py:55: AttributeError"}, "start": 1715624694279, "stop": 1715624694279, "uuid": "1fc68065-2743-4ab1-9d4e-6ab964b09463", "historyId": "99091ff4f3a54bbba7777a96f0a2fe5e", "testCaseId": "99091ff4f3a54bbba7777a96f0a2fe5e", "fullName": "tests.vwoLoginTests.test_vwo_login#test_vwo_login_positive", "labels": [{"name": "feature", "value": "TC#1 - VWO App Positive Test"}, {"name": "epic", "value": "VWO Login Test"}, {"name": "tag", "value": "positive"}, {"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwo_login"}, {"name": "host", "value": "DESKTOP-PUBF6P0"}, {"name": "thread", "value": "41456-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.test_vwo_login"}]}